Automattic/wp-calypso
# Site Migration: Show an error message when the site setup fails

Closes #90228

## Proposed Changes
* Show an error message guiding the user to  ask for support help
* Add tracking event when we don't have success to finish the prepare the site.



## Tech changes
Add configurations to manage the retry attempts and retry delay to optimize the test execution.

## Testing Instructions
It is not possible to test using calypso.live because it requires code changes to force errors.

* Go to the line 6 on the file https://github.com/Automattic/wp-calypso/blob/2a620b63cf341fa2a1e879e054a1954cf31ccee5/client/landing/stepper/hooks/use-prepare-site-for-migration.ts#L6
* Replace the plugin const with `
const PLUGIN = { name: 'migrate-guru-wrong/migrateguru-wrong', slug: 'migrate-guru' };
`
* Run the migration flow and check if the last step shows a screen like this:

<img width="822" alt="image" src="https://github.com/Automattic/wp-calypso/assets/38718/37206604-7954-4725-a2d3-64f5efcf6103">

## Pre-merge Checklist
